### ISSUE TITLE
Suggestion: add a nice and useful of output when print/summary of a grid object

### ISSUE THREAD

**Issue by matteodefelice** [SantanderMetGroup/loadeR#20]
Like for the `iris` Cubes (see this page: http://scitools.org.uk/iris/docs/latest/userguide/navigating_a_cube.html ), it would be nice to have useful information when using `print` or `summary` on a grid object. 
For example, several times I check the dimension of the grid object, the starting and ending date, so it would be nice to have those information easily without doing something like `str(grid$Data)` or `range(grid$Dates$start)`

**Comment by jbedia** [comment]
I agree. It is planned to move to S3/S4 at some point in the next months and overload methods summary, print etc... In the meantime, I suggest using `getShape`, `getSeason`,  `getDim` etc., which are quite handy. I take note of the suggestion for retrieving date ranges (I have missed it myself very often...)

**Comment by matteodefelice** [comment]
Thanks for the reply, but where are those functions (`getShape`, `getSeason`, etc.)?

**Comment by jbedia** [comment]
They are part of transformeR. See e.g. `?transformeR::getShape`

**Comment by matteodefelice** [comment]
great, I wasn't aware of those functions!

**Comment by jbedia** [comment]
There is also a `getRefDates` helper. All helper functions have been put in the [misc.R](https://github.com/SantanderMetGroup/transformeR/blob/devel/R/misc.R) file. I put in my to-do-list to document all these helpers in the same help file (gridHelpers? what do you think?).

**Comment by matteodefelice** [comment]
well, probably a meaningful file name could help. Thank you to point me out the existence of those useful functions!
